ios/AirPushNotification/CTAppController+APNAdditions.m

```diff
@@ -20,8 +20,11 @@
 @implementation CTAppController (APNAdditions)
 
 + (void)load {
-    
+    if([UNUserNotificationCenter class] != nil) {
```

Comment:
if this class exists, we're going to use it for everything instead of what's in the rest of this method which we use for < 10